test/microphysics2M_tests.jl

```diff
@@ -487,17 +487,19 @@ function test_microphysics2M(FT)
         )[2] ≈ 0 atol = eps(FT)
     end
 
-    TT.@testset "2M_microphysics - Seifert and Beheng 2006 effective radius" begin
+    TT.@testset "2M_microphysics - Seifert and Beheng 2006 radar reflectivity" begin
         #setup 
-        ρ_air = FT(1)
-        ρ_w = FT(1000)
-        q_liq = FT(3.654e-4)
-        N_liq = FT(22886304)
-        q_rai = FT(1.563e-4)
-        N_rai = FT(6813825)
+        # need to pass densities (ρ) in g/cm^3,
```

Comment:
Could we stick to passing in variables in base SI units? So meters and kilograms, etc. If we need to convert to different units for better accuracy, lets do it inside the reflectivity and effective radius functions